### ISSUE TITLE
Stuck at the firebase_app_distribution step

### ISSUE THREAD

**Issue by adrianchincm** [fastlane-community/fastlane-plugin-firebase_app_distribution#3]
Running my fastlane command in Android Studio terminal, I seem to be stuck in the 'Step : firebase_app_distribution' . However, the build is successfully uploaded into Firebase (I can see it in the console dashboard).

Below is my fastlane config : 
```
lane :distribute do

  changelog = prompt(
    text: "Changelog: ",
    multi_line_end_keyword: "END"
  )

  # build the release variant
  build_android_app(
    task: 'assemble',
    build_type: ‘Debug’,
    flavor: ‘staging’
  )

  # upload to firebase app distribution
  firebase_app_distribution(
              app: "1:xxxxxxxxx:android:xxxxxxxxxxx",
              groups: "android",
              release_notes: changelog,
              firebase_cli_path: "/usr/local/bin/firebase"
          )
end
```

**Comment by tagboola** [comment]
Hi Adrian! Tunde from the Firebase App Distribution team. Are you still seeing this issue? If so, can you send your console output?

**Comment by mamatagelanee07** [comment]
Facing same issue, 

```
################################################################################
  desc "Upload binary to Firebase App Distribution"
  ################################################################################
  lane :firebase do
        firebase_app_distribution(
                      app: "1:xxxxxxx:android:xxxxxxxxx",
                      testers: "xxxx@xxxx.xxx,
                      release_notes: "Lots of amazing new features to test out!",
                      firebase_cli_path: "/xxx/xxxxx/xx/firebase-tools",
                      apk_path: "./app/build/outputs/apk/debug/"
                  )
  end
```

Also, I can't see any files uploaded on Firebase console.

**Comment by KatieBarnett-Bilue** [comment]
I get the same, my apks are uploaded correctly but at the first firebase_app_distribution call the console stops outputting anything. I know my fastlane is still running though as subsequent commands succeed but nothing is output to the log as I would usually expect. 

Example from the console:
```
[02:53:50]: Cruising over to lane 'android uploadToFirebase' 🚖
--
  | [02:53:50]: ---------------------------------------
  | [02:53:50]: --- Step: firebase_app_distribution ---
  | [02:53:50]: ---------------------------------------
  | [02:53:55]: ▸

```
Example step:
```
  desc "Uploads the apk to firebase"
  private_lane :uploadToFirebase do |options|
    appId = options[:appId]
    firebase_app_distribution(
      app: appId,
      groups: 'xxx'
      firebase_cli_path: '/usr/local/bin/firebase',
      firebase_cli_token: 'xxxxxxx')
  end
```

**Comment by tagboola** [comment]
@KatieBarnett-Bilue debug functionality was added in version `0.1.3`. Can you run `fastlane update_plugins` to get the latest version, set the `debug` parameter to true as shown below, run the command and post your output?

```
 desc "Uploads the apk to firebase"
  private_lane :uploadToFirebase do |options|
    appId = options[:appId]
    firebase_app_distribution(
      app: appId,
      groups: 'xxx'
      firebase_cli_path: '/usr/local/bin/firebase',
      firebase_cli_token: 'xxxxxxx',
      debug: true)
  end
```

**Comment by mamatagelanee07** [comment]
> Facing same issue,
> 
> ```
> ################################################################################
>   desc "Upload binary to Firebase App Distribution"
>   ################################################################################
>   lane :firebase do
>         firebase_app_distribution(
>                       app: "1:xxxxxxx:android:xxxxxxxxx",
>                       testers: "xxxx@xxxx.xxx,
>                       release_notes: "Lots of amazing new features to test out!",
>                       firebase_cli_path: "/xxx/xxxxx/xx/firebase-tools",
>                       apk_path: "./app/build/outputs/apk/debug/"
>                   )
>   end
> ```
> 
> Also, I can't see any files uploaded on Firebase console.

Mine worked in Mac terminal, It was just stuck in Android Studio Terminal,
Thanks

**Comment by rebehe** [comment]
This is Rebecca, engineer on App Distribution - thanks all for the awesome feedback and detailed info! We're working on it and will update you when we have a fix.

**Comment by KatieBarnett-Bilue** [comment]
I added the debug flag and got the same output:
```
[02:09:30]: Cruising over to lane 'android uploadToFirebase' 🚖
--
  | [02:09:30]: ---------------------------------------
  | [02:09:30]: --- Step: firebase_app_distribution ---
  | [02:09:30]: ---------------------------------------
  | [02:09:34]: ▸
```

**Comment by rebehe** [comment]
Hi all, we've isolated the problem to be specifically if you're running the standalone binary version of the Firebase CLI https://firebase.google.com/docs/cli through the Android Studio terminal. The team is planning on updating the binaries early next week to fix this. Until then, a temporary workaround is to use a different terminal, or by using the npm installed version of the Firebase CLI.

**Comment by tonybaroneee** [comment]
A fix for this has been released in [v7.6.0](https://github.com/firebase/firebase-tools/releases/tag/v7.6.0) of the Firebase CLI standalone binary versions. Thanks for the reports!
